### PR TITLE
ACM-1958 Clarify step

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by jelkosz** [comment]
lgtm

File: clusters/cluster_lifecycle/create_infra_env.adoc [hunk 1]
```diff
@@ -38,7 +38,9 @@ The CIM service is provided with the {mce} and deploys {ocp-short} clusters. CIM
 
 To enable the CIM service, complete the following steps: 
 
-. If your hub cluster platform is bare metal, Red Hat OpenStack Platform, VMware vSphere, or was installed using the user-provisioned infrastructure (UPI) method and the platform is `None`, modify the `Provisioning` resource to allow the Bare Metal Operator to watch all namespaces by running the following command:
+. This step only applies if your hub cluster is installed on one of the following platforms: bare metal, Red Hat OpenStack Platform, VMware vSphere, or was installed using the user-provisioned infrastructure (UPI) method and the platform is `None`. Skip this step if your hub cluster is on any other platform.
```
Comment (oafischer) [inline clusters/cluster_lifecycle/create_infra_env.adoc]:
```suggestion
. This step only applies if your hub cluster is installed on one of the following platforms: bare metal, Red Hat OpenStack Platform, VMware vSphere, or was installed by using the user-provisioned infrastructure (UPI) method and the platform is `None`. Skip this step if your hub cluster is on any other platform. 
```

**Review by oafischer — APPROVED** [review]
lgtm, one small suggestion

**Comment by cadawson** [comment]
Good suggestion. Thanks OIiver.

**Pull request merged**